include/stdint.h

```diff
@@ -273,8 +273,13 @@ typedef _uint64_t           uint_fast64_t;
 
 /* Integer types capable of holding object pointers */
 
+#ifdef __LP64__
+typedef long                intptr_t;
+typedef unsigned long       uintptr_t;
+#else
 typedef _intptr_t           intptr_t;
 typedef _uintptr_t          uintptr_t;
+#endif
 
```

Comment:
I don't believe that this is a correct change.  You chould not put any CPU specific definitions in the POSIX standard stdint.h.  That is the wrong way to go.

Instead, you need to change arch/risc-v/include/types.h.  That is the appropriate place for this change.

Reply:
@patacongo Thanks for the comment. I understand what I should do. I'll fix the commits and force to push again.